xbmc/windowing/X11/GLContextEGL.cpp

```diff
@@ -17,17 +17,22 @@
  *  <http://www.gnu.org/licenses/>.
  *
  */
+#include "system_gl.h"
```

Comment:
please use system.h. I will terminate system_gl.h in some future task because I consider it crap.